src/modules/control_allocator/module.yaml

```diff
@@ -30,6 +30,7 @@ parameters:
                 10: Helicopter (tail ESC)
                 11: Helicopter (tail Servo)
                 12: Helicopter (Coaxial)
+                13: Rover (Mecanum)
```

Comment:
You currently just use it (the CA module) for the visualization of the motors in the ground station? Wonder if it's worth adding it here for that - could lead folks to think that the motor allocation is handled by the CA module.

Reply:
I missed the Ackermann and Differential rover types higher up - disregard this comment.